### ISSUE TITLE
PUBG Mobile touch problem

### ISSUE THREAD

**Issue by ArifAGA98** [ananjaser1211/FloydQ_Reports#194]
Phone touch does not detect more than two fingers.

**Comment by ananjaser1211** [comment]
This is a hardware issue or replaced touch panel.

~10 fingers show up in multitouch test, and work fine with PUBG

**Comment by ArifAGA98** [comment]
Yes, I renewed the screen of the phone with the original screen, but this
touch problem was not available.  This issue happened after the update to
pubc (1.0). Thank you for feedback.  I am grateful to you for your work.

19 Eyl 2020 Cmt 12:36 tarihinde AnanJaser <notifications@github.com> şunu
yazdı:

**Comment by ananjaser1211** [comment]
it may be possible that your touch screen firmware was only designed to work on certain firmwares. similar case happened on J series devices, i had a replacement screen that never worked beyond nougat and was only 2 point multi touch

i have tested PUBG today to debug this and the dolby issue you mentioned. did not notice issues with touch. so sadly it seems that its a H/W related issue/incompatibility

**Comment by ArifAGA98** [comment]
When this problem occurred in the game, I installed lineage os 17.1 and the
problem was fixed.  Later, a small package update came to the game and it
was the same problem again as in Floyd v4.  So I uninstalled the game and
reinstalled it and the problem was fixed.  I think this is a problem with
my phone or game. I will install Floyd v4 again to try this.

20 Eyl 2020 Paz 02:05 tarihinde AnanJaser <notifications@github.com> şunu
yazdı:
